app/src/main/res/values/strings.xml

```diff
@@ -642,15 +642,15 @@
     <string name="error_timeout">Connection timeout</string>
     <string name="error_download_resource_gone">Cannot recover this download</string>
     <string name="clear_download_history">Clear download history</string>
-    <string name="confirm_prompt">Do you want to clear your download history or delete all downloaded files?</string>
+    <string name="confirm_prompt">Clear your download history or delete all downloaded files?</string>
```

Comment:
I'd rather keep the "Do you want to", it makes it clearer that you can choose between two options

Reply:
```suggestion
    <string name="confirm_prompt">Clear your download history, or delete all downloaded files?</string>
```

Reply:
"Do you want to" is redundant. It adds text to the whole experience.
Added the comma to make it clear.